tests/cross/src/test/scala/tests/pc/CompletionExtensionSuite.scala

```diff
@@ -0,0 +1,28 @@
+package tests.pc
+
+import tests.BaseCompletionSuite
+import tests.BuildInfoVersions
+
+class CompletionExtensionSuite extends BaseCompletionSuite {
+
+  override def requiresJdkSources: Boolean = true
+
+  override def excludedScalaVersions: Set[String] =
+    BuildInfoVersions.scala3Versions.dropWhile(_ != "3.0.0-M2").toSet
+
+  check(
```

Comment:
I think this is unrelated to the PR, could you maybe add the test in a separate one?